src/peergos/server/Downloader.java

```diff
@@ -13,6 +14,7 @@
 import java.util.function.*;
 
 public class Downloader {
+	private static final Logger LOG = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
```

Comment:
why not Logger.getGlobal?

Reply:
Sure, feel free to update it

Reply:
I don't have time for that. I barely have time to manually review every change here.